pkg/agent/core/ngt/handler/grpc/index_test.go

```diff
@@ -62,6 +62,26 @@ func Test_server_CreateIndex(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		- Equivalence Class Testing
+			- case 1.1: success to create index with 1 uncommitted insert index
+			- case 1.2: success to create index with 100 uncommitted insert index
+			- case 2.1: success to create index with 1 uncommitted delete index
+			- case 2.2: success to create index with 100 uncommitted delete index
+			- case 3.1: success to create index with 1 uncommitted insert + delete index
+			- case 3.2: success to create index with 100 uncommitted insert + delete index
+
+		- Boundary Value Testing
+			- case 1.1: success to create index with 0 uncommitted index
+
+		- Decision Table Testing
+		    // with uncommitted index count 100
+			- case 1.1: success to create index with poolSize > uncommitted index count
+			- case 1.2: success to create index with poolSize < uncommitted index count
+			- case 1.3: success to create index with poolSize = uncommitted index count
+			- case 1.4: success to create index with poolSize = 0
+	*/
```

Comment:
since the target of this test is not testing the insert / delete request, but the create index API, I think there is no way to check the dimension size of the vector.

Is there any other factor that you are concerning about in this test?

Reply:
If the error may occur, it would be better to test

Reply:
but how to test the dimension size in create index test?

Reply:
It's your business. lol
You don't have to when geting a panic error

Reply:
Sorry I couldn't think of any test cases about index dimension in createIndex test, please let me know if you have any idea.